shop/src/Components/Price/index.jsx

```diff
@@ -1,20 +1,81 @@
 import "./Price.scss";
-const Price = () => {
+import { useState, useEffect } from "react";
+import { dolorFormat } from "../../util/dolarFormat";
+
+const Price = ({ shoeItems, step, handleOnPriceRange }) => {
+  const [minPrice, setMinPrice] = useState();
+  const [maxPrice, setMaxPrice] = useState();
+
+  const [minValue, setMinValue] = useState(0);
+  const [maxValue, setMaxValue] = useState(30000);
+
+  const getMinPrice = () => {
+    setMinPrice(Math.min(...shoeItems.map((shoe) => shoe.retail_price_cents)));
+  };
+
+  const getMaxPrice = () => {
+    setMaxPrice(Math.max(...shoeItems.map((shoe) => shoe.retail_price_cents)));
+  };
+
```

Comment:
same for `getMinPrice` and `getMinPrice`. we can pass in a param to lets the function what we are looking for